test/unit/reductions/exponentiated_gradient/test_exponentiatedgradient_smoke.py

```diff
@@ -100,7 +106,7 @@ def test_smoke(self, testdata):
 
     def test_simple_fit_predict(self):
         estimator = LeastSquaresBinaryClassifierLearner()
-        constraints = DemographicParity()
+        constraints = TruePositiveRateDifference()
```

Comment:
You can parametrize this test to use all the constraints, but don't replace the existing one with DP, please.

Reply:
Sorry about this, again. Yeah, I will just parameterize it.

Reply:
Done 👍